website/docs/docs/deploy/deploy-jobs.md

```diff
@@ -13,57 +13,63 @@ You can use deploy jobs to build production data assets. Deploy jobs make it eas
 - Job run details, including run timing, [model timing data](#model-timing), and [artifacts](/docs/deploy/artifacts)
 - Detailed run steps with logs and their run step statuses
 
-You can create a deploy job and configure it to run on [scheduled days and times](#schedule-days) or enter a [custom cron schedule](#custom-cron-schedules).
+You can create a deploy job and configure it to run on [scheduled days and times](#schedule-days) or enter a [custom cron schedule](#cron-schedule).
 
 
 ## Prerequisites
 
-- You must have a dbt Cloud account and [Developer seat license](/docs/cloud/manage-access/seats-and-users). If you don't, you can [sign up](https://www.getdbt.com/signup/) for a [free account](https://www.getdbt.com/pricing/). 
+- You must have a [dbt Cloud account](https://www.getdbt.com/signup/) and [Developer seat license](/docs/cloud/manage-access/seats-and-users).
+    - For the [Trigger on job completion](#trigger-on-job-completion) feature, your dbt Cloud account must be on the [Team or Enterprise plan](https://www.getdbt.com/pricing/).
 - You must have a dbt project connected to a [data platform](/docs/cloud/connect-data-platform/about-connections).
 - You must have [access permission](/docs/cloud/manage-access/about-user-access) to view, create, modify, or run jobs.
 - You must set up a [deployment environment](/docs/deploy/deploy-environments). 
 
 ## Create and schedule jobs {#create-and-schedule-jobs}
 
-1. On your deployment environment page, click **Create Job** > **Deploy Job** to create a new deploy job. 
-2. Options in the **Job Description** section:
-    - **Job Name** &mdash; Specify the name for the deploy job. For example, `Daily build`.
+1. On your deployment environment page, click **Create job** > **Deploy job** to create a new deploy job. 
+2. Options in the **Job settings** section:
+    - **Job name** &mdash; Specify the name for the deploy job. For example, `Daily build`.
+    - (Optional) **Description** &mdash; Provide a description of what the job does (for example, what the job consumes and what the job produces). 
```

Comment:
Don't we usually put this after?
```suggestion
    - **Description** (optional) &mdash; Provide a description of what the job does (for example, what the job consumes and what the job produces). 
```

Reply:
actually, i'm not sure. i don't see that in the style guide. 

i'm accustomed to before. that way, people can move one faster if they're ok with the defaults